### PR TITLE
log all DDL statements

### PR DESCRIPTION
**Pull request by heavycrystal**

### CODE REVIEW

File: flow/connectors/postgres/client.go [hunk 1]
```diff
@@ -620,3 +622,11 @@ func (c *PostgresConnector) ExecuteCommand(ctx context.Context, command string)
 	_, err := c.conn.Exec(ctx, command)
 	return err
 }
+
+func (c *PostgresConnector) execWithLogging(ctx context.Context, query string, tx pgx.Tx) (pgconn.CommandTag, error) {
```
Comment (serprex) [inline flow/connectors/postgres/client.go:626]: split into two separate methods, one taking tx, one not

File: flow/connectors/clickhouse/clickhouse.go [hunk 2]
```diff
@@ -216,3 +217,11 @@ func (c *ClickhouseConnector) ConnectionActive(ctx context.Context) error {
 	// This also checks if database exists
 	return c.database.PingContext(ctx)
 }
+
+func (c *ClickhouseConnector) execWithLogging(ctx context.Context, query string, tx *sql.Tx) (sql.Result, error) {
```
Comment (serprex) [inline flow/connectors/clickhouse/clickhouse.go:221]: same, split

File: flow/connectors/snowflake/snowflake.go [hunk 3]
```diff
@@ -604,28 +606,19 @@ func (c *SnowflakeConnector) CreateRawTable(ctx context.Context, req *protos.Cre
...
 
 	if !schemaExists.Valid || !schemaExists.Bool {
-		_, err := c.database.ExecContext(ctx, fmt.Sprintf(createSchemaSQL, c.rawSchema))
+		_, err := c.execWithLogging(ctx, fmt.Sprintf(createSchemaSQL, c.rawSchema), nil)
 		if err != nil {
 			return nil, err
 		}
 	}
-
-	createRawTableTx, err := c.database.BeginTx(ctx, nil)
```
Comment (serprex) [inline flow/connectors/snowflake/snowflake.go:613]: why removing tx?

File: flow/connectors/snowflake/snowflake.go [hunk 4]
```diff
@@ -855,3 +848,11 @@ func (c *SnowflakeConnector) CreateTablesFromExisting(ctx context.Context, req *
 		FlowJobName: req.FlowJobName,
 	}, nil
 }
+
+func (c *SnowflakeConnector) execWithLogging(ctx context.Context, query string, tx *sql.Tx) (sql.Result, error) {
```
Comment (serprex) [inline flow/connectors/snowflake/snowflake.go:852]: split

File: flow/connectors/snowflake/snowflake.go (hunk 3 above)
Comment (heavycrystal) [reply flow/connectors/snowflake/snowflake.go:613]: Snowflake doesn't have transactional DDL, and this tx is for a single statement anyway

**Review by serprex — APPROVED** [review]

**Pull request merged**